02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,54 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title></title>
-</head>
-<body>
-</body>
+<html style= "background: lightblue" >
+  <head>
+    <title>Jonathan Gleason</title>
+  </head>
+  <body>
+    <h1>Jonathan Gleason
+    </h1>
+    <h5><em>2101 Burton Dr. Austin, TX 78741</em>
```

Comment:
Indentation: Pub block elements on their own line and indented 2 spaces from their parent element. Makes it easier to find errors when the code gets more complex.